tests/integrationtest/r/ddl/db_read_only.result

```diff
@@ -0,0 +1,114 @@
+alter database db_read_only set tiflash replica 1;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+drop database db_read_only;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+create table t1(c1 int);
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+create table t2 like t;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+create table pt (a int, b int) partition by hash(a) partitions 4;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t add column cc int not null;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t add index (a);
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+create index c1 on t(a);
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t alter index b invisible;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t change column c c varchar(10);
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t drop column c;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t drop index b;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+drop index b on t;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t modify column c varchar(10);
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t rename index b to b_1;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t drop foreign key fk_a;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t add foreign key fk_a1(b) references s(a);
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t add foreign key fk_a2(b) references db_read_only_1.s(a);
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table t partition by hash(a) partitions 4;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table members drop partition p1990;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table members truncate partition p1980;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table members add partition (partition `p1990to2010` values less than (2010));
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter table members reorganize partition `p1990to2010` into
+(partition p1990 values less than (2000),
+partition p2000 values less than (2010),
+partition p2010 values less than (2020),
+partition p2020 values less than (2030),
+partition pMax values less than (maxvalue));
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+rename table t to t_old;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+rename table db_read_only.t to db_read_only_1.t;
+Error 3809 (HY000): Schema 'db_read_only' is in read only mode.
+alter schema db_read_only read only 0;
+alter schema db_read_only_1 read only 1;
```

Comment:
the grammer is a little odd, the `0` is redundant, and the `read only 1` can be `read write` to be more readable

Reply:
Can't agree more, but to keep the syntax consistent with [mysql](https://dev.mysql.com/doc/refman/8.4/en/alter-database.html#alter-database-read-only), we implement in the same syntax. Maybe in the future, we can achieve `read only` and `read write` on this basis.